autodetect-dependabot.jl

```diff
@@ -0,0 +1,33 @@
+module AutodetectDependabot
+
+function get_possible_branch_names()
```

Comment:
None of this is necessary if it's only for GHA.

Reply:
Good catch. I'll fix that now.

Reply:
Alright I've removed the Travis and AppVeyor stuff.